apps/src/templates/studioHomepages/SectionsTable.jsx

```diff
@@ -93,10 +94,9 @@ const SectionsTable = React.createClass({
     sections: shapes.sections,
     isRtl: PropTypes.bool.isRequired,
     isTeacher: PropTypes.bool.isRequired,
-    codeOrgUrlPrefix: PropTypes.string.isRequired,
     canLeave: PropTypes.bool.isRequired,
-    updateSections: PropTypes.func.isRequired,
-    updateSectionsResult: PropTypes.func.isRequired
+    updateSections: PropTypes.func,
+    updateSectionsResult: PropTypes.func
```

Comment:
These props are only used for the student view of the sections table (for the "Leave section" controls), therefore they shouldn't be required in the teacher view.